integrations/zendesk/src/definitions/index.ts

```diff
@@ -0,0 +1,44 @@
+import type { IntegrationDefinitionProps } from '@botpress/sdk'
+import { z } from 'zod'
+
+export { actions } from './actions'
+export { events } from './events'
+export { channels } from './channels'
+
+export const configuration = {
+  schema: z.object({
+    baseURL: z.string({
+      description: 'Your zendesk organization URL. e.g. https://{subdomain}.zendesk.com',
+    }),
+    username: z.string({
+      description: 'Your zendesk account email, add "/token" to the end. e.g. jdoe@example.com/token',
+    }),
+    apiToken: z.string({
+      description: 'Zendesk API Token',
+    }),
+  }),
+}
+
+export const states: IntegrationDefinitionProps['states'] = {
```

Comment:
**nitpick:** use satisfies instead. The compiler will infer the type of your states and could catch more errors (ex: if your `ui` field describes fields that are not in the `shema`)